docs/references/api.md

```diff
@@ -75,6 +75,28 @@ except ImportError:
 
 You could do something similar by catching certain errors when using the `pro api` subcommand, but there are more cases that could indicate an old version, and it generally isn't recommended.
 
+
+### Errors and Warnings fields
+
+When using the API through the CLI, we use two distinct fields to list issues to the users, *errors*
```

Comment:
```suggestion
When using the API through the CLI, we use two distinct fields to list issues to the users; *errors*
```

Reply:
Shouldn't this be a ":" instead ?

Reply:
Normally you'd use a colon when introducing a list of objects (which would be separated by semicolons), but in this case, since these list entries are joined with a conjunction it's more usual to introduce them with a semicolon instead.

Reply:
Got it. Thanks for the explanation @s-makin